doc/source/ray-core/scheduling/resources.rst

```diff
@@ -21,7 +21,7 @@ Physical resources are resources that a machine physically has such as physical
 and logical resources are virtual resources defined by a system.
 
 Ray resources are **logical** and don’t need to have 1-to-1 mapping with physical resources.
-For example, you can start a Ray head node with 3 GPUs via ``ray start --head --num-gpus=3`` even if it physically has zero.
+For example, you can start a Ray head node with 0 logical CPUs via ``ray start --head --num-cpus=0`` even if it physically has eight.
```

Comment:
This signals the Ray scheduler to not schedule any tasks or actors on the head node, as you have other Ray-related processes running that may demand resources.